docs/basics/userguide/seeding.md

```diff
@@ -320,8 +351,11 @@ random sample 2 from seeder2 rng:  -0.1930990650226178
 </br>
 
 ## External libraries
-You can also use a seeder to seed external libraries (such as torch) using the method `set_external_seed`.
-It will be usefull if you want reproducibility with external libraries. In this example, you will use `torch` to generate random numbers.
+You can also use a seeder to seed some external libraries using the method `set_external_seed`.
+(currently only [torch](https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html))
+
+
+It will be useful if you want reproducibility with external libraries. In this example, we will use `torch` to generate random numbers.
 
 If you run this code many time, you should have different outputs.
```

Comment:
Not useful to say it, you show it just below

Reply:
I think it's useful to point specificaly on this information.